### DISCUSSION TITLE
Heading 2

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#318] [Component suggestions]
<img width="720" alt="visuele weergave van de heading 2 component" src="https://github.com/user-attachments/assets/2f8e6322-f2f2-4196-8ee7-7c00895af18c">

## Naam

Heading 2

## Doel
Koptekst die volgt na de hoofdkop en in de koppenstructuur op het tweede niveau staat.

## Ook bekend als

Heading, Titel

## Gerelateerde componenten

[Heading](https://github.com/orgs/nl-design-system/discussions/220), [Heading 1](https://github.com/orgs/nl-design-system/discussions/317), [Heading 3](https://github.com/orgs/nl-design-system/discussions/319), [Heading 4](https://github.com/orgs/nl-design-system/discussions/320), [Heading 5](https://github.com/orgs/nl-design-system/discussions/321), [Heading 6](https://github.com/orgs/nl-design-system/discussions/322), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Heading 2 op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=heading+2)
